Clients/GUI/Shared/NewsComponent.razor

```diff
@@ -0,0 +1,109 @@
+﻿@page  "/NewsComponent"
+
+    <div class="container">
+        @if (errorMessage == "")
+        {
+            <h1>@newsEconomics[0].Category</h1>
+            <div class="row row-margin-bottom">
+                @for (int i = 0; i < 2; ++i)
+                {
+                    var newsItem = newsEconomics[i];
+                    <div class="col-md-6">
+                        <div class="lib-panel">
+                            <div class="row box-shadow">
+                                <div class="lib-row lib-header">
+                                    <a href="@newsItem.Link">@newsItem.Title</a>
+                                    <div class="lib-header-seperator"></div>
+                                </div>
+                                <div class="lib-row lib-desc">
+                                    <p><span class="glyphicon glyphicon-calendar" aria-hidden="true"></span> @newsItem.PubDate</p>
+                                    <p>@newsItem.Description</p>
+                                    <p>@newsItem.Author</p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                }
+            </div>
+
+            <h1>@newsMarkets[2].Category</h1>
```

Comment:
This looks like copypaste (i mean same code for each category)

Reply:
> This looks like copypaste (i mean same code for each category)

I mean that I suggest to do like
```
foreach (category)
{
    some html and c# code for each category block
}
```

Reply:
I am also thinking about this decision

Reply:
Fixed